src/Werte/Altersvorsorge/Produkterfahrung.php

```diff
@@ -24,18 +22,15 @@ public function __construct()
     }
 
     /**
-     * @param int $id
-     *
-     * @return ProdukterfahrungInterface
-     * @throws \Demv\Werte\Exception\EntryNotFoundException
+     * @throws EntryNotFoundException
      */
     public function getOne(int $id): ProdukterfahrungInterface
     {
         return parent::getOne($id);
     }
 
     /**
-     * @return ProdukterfahrungInterface[]
+     * @return ProdukterfahrungInterface[]|ValueInterface[]
```

Comment:
ist das gewollt so?
In diesem Fall ist `ProdukterfahrungInterface` ja genauer, wieso daher ValueInterface?

Reply:
Tatsächlich habe ich damit nur die Tools Happy gemacht.

Ja `ProdukterfahrungInterface` ist genauer. Aber es ist im Programmablauf nicht 100%ig klar, dass es nicht auch ein ValueInterface sein kann. (die Architektur ist hier leider nicht so gutgewählt)

Wir können die entsprechenden phpstan-Fehler aber auch ignorieren.